data/src/main/java/com/linkedin/data/DataMap.java

```diff
@@ -110,10 +112,27 @@ public DataMap(int initialCapacity, float loadFactor)
     super(initialCapacity, loadFactor, _checker);
   }
 
+  /**
+   * Constructs a {@link DataMap} backed by the given specific map.
+   *
+   * <p>This constructor is meant to be invoked only from code-generated models.</p>
+   *
+   * @param specificMap provides the specific map.
+   *
+   * @see HashMap
```

Comment:
This probably doesn't make sense anymore.

Reply:
Why not? Without this how do we call the placeholder no copy constructor in CheckedMap?